packages/wallet-core/src/store/actions/updateNFTs.ts

```diff
@@ -36,6 +36,19 @@ export const updateNFTs = async (
       const nfts = nftAssetsFetched.map((nftAsset: NFT) => {
         const nftAssetStoredInState = nftAssetsStoredInState.find((asset: NFT) => asset.token_id === nftAsset.token_id);
         const starred = nftAssetStoredInState ? nftAssetStoredInState.starred : false;
+        const replaceIPFSUrl = (url: string) => {
```

Comment:
Wasn't this supposed to go in chainify? 
https://github.com/liquality/chainify/pull/612

Reply:
I have moved it there